nextflow.config

```diff
@@ -9,7 +9,7 @@ process {
     }
     
     withName: RunAlmanack {
-        container = 'community.wave.seqera.io/library/pip_almanack:34e622e308f4a975'
+        container = 'python:3.11-slim'
```

Comment:
you now have a double specified container. Both here and in the process. For process specific containers only specify once, recommend doing so in the process